scripts/configure-multi-node-aio.yml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
I reused bootstrap-aio.yml because previously @hughsaunders added the ability to set the variable TARGET to set configurations for mnaio (https://github.com/rcbops/rpc-openstack/blob/master/scripts/bootstrap-aio.yml#L131-L141). I'm using also TARGET to skip the "Execute the OSA AIO bootstrap". I could add the final tasks to create user_osa_variables_overrides.yml to bootstrap-aio.yml and have them run conditionally as well.

Though I guess the real issue is that boostrap-aio is doing more than just bootstrapping AIOs. We could split the configuration parts and the actual bootstrap part into separate playbooks and only use the config playbook for MNAIO.